qiita_pet/handlers/portal.py

```diff
@@ -0,0 +1,41 @@
+from tornado.web import authenticated, HTTPError
+
+from qiita_db.study import Study
+from qiita_db.portal import Portal
+from .base_handlers import BaseHandler
+
+
+class StudyPortalHandler(BaseHandler):
+    def check_admin(self):
+        if self.current_user.level != "admin":
+            raise HTTPError(403, "%s does not have access to portal editing!" %
+                            self.current_user.id)
+
+    def render_page(self):
+        # COLS MUST HAVE AN ID COLUMN!
```

Comment:
I don't understand what are you trying to say (I also don't understand why you're yelling either hehehe)